common/meta/types/src/database.rs

```diff
@@ -35,12 +35,20 @@ pub struct DatabaseInfo {
     pub meta: DatabaseMeta,
 }
 
```

Comment:
`common/meta/types` will move to `query` in final?

Reply:
Metadatas are defined in two crate: common/meta/types and common/datavalues.

It looks like the database, table related types should be kept in the same crate.
And IMHO, to be more flexible, e.g., in case there are other crates that need these types, leaving these metadata types outside crate `query` would be better, if possible.

And there are still some metasrv-related types in common/meta/types.

Reply:
> leaving these metadata types outside crate query would be better, if possible.

Agree.
So leave them still in the `common/meta/types` or a new crate? Prefered sperate them from `common/meta/types` to a new crate, to keep the `common/meta/types` only for `metasrv-related types`.

Reply:
A new crate sounds good:)